.github/workflows/hw-event-proxy.yaml

```diff
@@ -23,7 +23,8 @@ jobs:
           skip-pkg-cache: true
           skip-build-cache: true
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
-          version: v1.42.1
+          version: v1.42
+          args: --timeout 3m0s
```

Comment:
isn't 3m too long? is this timeout to download to run linter?

Reply:
I searched around and 3m seems to be a common solution. It is the timeout for the execution. 
https://github.com/golangci/golangci-lint-action/issues/297